hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningServiceTest.java

```diff
@@ -231,6 +237,63 @@ public void run()
         }, 2);
     }
 
+
+    @Test
+    public void testExecutionShouldNotTimeoutIfTraceable_WithMultiCallback() throws Exception {
```

Comment:
Thats right @Donnerbart,  but this particular test seems not `quick` at first glance. There is a very long sleep `Thread.sleep(15000)`.